src/components/registration/FamilyDetailsSidebar.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Drawer } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+
+const drawerWidth = 250;
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: "flex",
+  },
+  h3: {
+    padding: "20px",
+    width: drawerWidth,
+  },
+  drawer: {
+    width: drawerWidth,
+  },
+}));
+
+function FamilyDetailsSidebar({ isOpen, rowData, handleClose }) {
+  const classes = useStyles();
+  const firstName = rowData[0];
+  const lastName = rowData[1];
+  return (
+    <Drawer
+      anchor="right"
+      variant="temporary"
+      className={classes.drawer}
+      open={isOpen}
+      onClose={handleClose}
+    >
+      <h3 className={classes.h3}>
+        Hey {firstName} {lastName}!
+      </h3>
+    </Drawer>
+  );
+}
+
+FamilyDetailsSidebar.defaultProps = {
+  isOpen: false,
+  rowData: ["Test", "User"],
+  handleClose: () => {},
+};
```

Comment:
is it valid to optionally pass in these props? if not, we may be ok with raising an exception instead of providing default value

Reply:
eslint was complaining about it, so i made the prop required!